wpimath/src/test/java/edu/wpi/first/math/filter/DebouncerTest.java

```diff
@@ -2,53 +2,65 @@
 // Open Source Software; you can modify and/or share it under the terms of
 // the WPILib BSD license file in the root directory of this project.
 
-package edu.wpi.first.wpilibj;
+package edu.wpi.first.math.filter;
 
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-import edu.wpi.first.wpilibj.simulation.SimHooks;
+import edu.wpi.first.util.WPIUtilJNI;
 import org.junit.jupiter.api.Test;
 
 class DebouncerTest {
   @Test
   void debounceRisingTest() {
     var debouncer = new Debouncer(0.02, Debouncer.DebounceType.kRising);
 
+    WPIUtilJNI.enableMockTime();
```

Comment:
This will break other tests unless it's disabled (use setup/teardown).

Reply:
afaict there is no disable?

Reply:
Added in #3839.